contracts/interfaces/Integrations.sol

```diff
@@ -39,12 +39,12 @@ interface ISafeProtocolStaticFallbackMethod {
 }
 
 /**
- * @title ISafeProtocolGuard - An interface that a Safe guard should implement.
- * @notice In Safe protocol, a guard can approve or deny transactions based on the logic it implements.
+ * @title ISafeProtocolHook - An interface that a Safe hook should implement.
+ * @notice In Safe protocol, a hook can approve or deny transactions based on the logic it implements.
  */
-interface ISafeProtocolGuard is IERC165 {
+interface ISafeProtocolHook is IERC165 {
```

Comment:
I used the plural in the specs, as this interface specified 3 hooks.

Reply:
For now only updating to `ISafeProtocolHooks` in this PR.